doc/examples/applications/plot_tool_using_holoviz.py

```diff
@@ -0,0 +1,202 @@
+"""
+==========================================================
+Explore and annotate image data interactively with HoloViz
+==========================================================
+
+In this tutorial, we build interactive tools and an application using
+`Panel <https://panel.holoviz.org>`_ and `HoloViews <https://holoviews.org>`_.
+
+We will build an application based on the Sobel algorithm to interactively
+find a great color map for the visualization. We will add some nice features,
+like a download button, to make sharing your findings easier.
+
+These techniques can make your analyses more interactive and, hence, powerful.
+
+.. figure:: https://user-images.githubusercontent.com/42288570/146666639-02f0106f-a3e2-4306-9033-707554e4e61e.gif
+   :alt: Screencast showcasing the example application.
+
+   Screencast showcasing the example application.
+
+
+You can develop using notebooks or .py files. You can use it with a very
+broad range of Pythons viz. You can use the tools in a notebook, your 
+editor/ IDE or deploy to a web server.
+
+To run this example you would need to get some python packages
+installed. You can create an environment from scratch using.
+
+.. code:: bash
+
+   conda create -n scikit-image-demo -c conda-forge jupyterlab panel holoviews scikit-image selenium firefox geckodriver
+   conda activate scikit-image-demo
+   jupyter serverextension enable panel.io.jupyter_server_extension
+"""
+
+
+from io import BytesIO
+
+import holoviews as hv
+import panel as pn
+from skimage import data, filters
+
+pn.extension(sizing_mode="fixed")
+hv.extension("bokeh")
+
+
+#######################################
+# Load an image of coins and display it
+
+image = data.coins()
+bounds = (-1, -1, 1, 1)
+height, width = image.shape
+aspect_ratio = float(width) / float(height)
+
+height = 400
+width = int(height * aspect_ratio)
+
+hv.Image(image, bounds=bounds).opts(
+    height=height,
+    width=width,
+    cmap="binary_r",
+    title="Before",
+    active_tools=["box_zoom"],

```

Comment:
@MarcSkovMadsen could we do without option 'active_tools' so that we could use a backend other than 'bokeh' and, hence/perhaps, have the figure display in the static HTML page?